example/cdp.js

```diff
@@ -0,0 +1,68 @@
+const _ = require("lodash");
```

Comment:
add `.npmignore` file with 

```
node_modules
example
```

to avoid publishing examples

Reply:
https://github.com/Kava-Labs/javascript-sdk/pull/20/commits/c64635c825e8c08f6ff972d7a36abe852a75c2c4